### ISSUE TITLE
Wurst 1.8 and 1.11 crash

### ISSUE THREAD

**Issue by LukasD666** [Wurst-Imperium/Wurst-MC-1.8#2442]
When I try to start the game with wurst 1.8 this message pops up and the game doesn't start.

[12:53:01] [Client thread/INFO]: Setting user: xXAle_ShishXx
[12:53:01] [Client thread/INFO]: (Session ID is token:1337535510N:c2bcfb7a0a6c3259b72a7e864cfd1a59)
[12:53:02] [Client thread/INFO]: LWJGL Version: 2.9.1

In 1.11 version the massage is a little bit different.

[12:48:32] [Client thread/INFO]: Setting user: xXAle_ShishXx
[12:48:32] [Client thread/WARN]: Skipping bad option: streamPreferredServer:
[12:48:32] [Client thread/INFO]: LWJGL Version: 2.9.4

Any solutions?

**Comment by 1dkoz** [comment]
Nope. But don't share your session ID when you're playing minecraft.
